css/style.css

```diff
@@ -16,4 +16,129 @@ body {
  * - When using Flexbox, remember the items you want to move around need to be inside a parent container set to 'display: flex'
  */
 
+/* This is Header (Navigation & Hero )*/
+
+/* Navigation */
+nav{
+    display:flex;
+    justify-content: center;
+    align-items: center;
+    height:140px;
+    width: 80%;
+    margin: auto;
+    font-size: 2.5rem;
+    }
+
+a{
+    height: 60px;
+    margin-left: 80px;
+    text-decoration: none;
+    color: rgb(65, 63, 63);
+    }
+
+a:hover{
+    color: rgb(250, 115, 4);
+    }
+
+.logo{
+    width:50px;
+    height:auto;
+    margin-right: auto;
+    margin-left: 0;
+    }
+
+.logo img{
+    width: 100%;
+    height: auto;
+    }
+
+ /* Hero style */
+.hero-image{
+    background-image: url(../img/first-background.jpg);
+    background-position: center;
+    background-repeat: no-repeat;
+    background-size: cover;
+    width: 100%;
+    height: 900px;
+    position:relative;
+}
+
+.hero-text{
+    text-align:center;
+    position: absolute;
+    color: white;
+    top:20%;
+    left:40%;
+}
+
+.hero-text h1{
+    font-size: 5rem;
+}
+
+.hero-text p{
+    font-size: 3rem;
+}
+
+button {
+    background-color: #f2750e; /* Orange */
+    border: none;
+    border-radius: 5px;
+    color: white;
+    font-size: 2rem;
+    padding: 20px 30px;
+    text-align: center;
+    text-decoration: none;
+  }
+
+/*This is main*/
+
+main h2{
+    margin-top: 100px;
+    font-size:4rem;
+    color: grey;
+    text-align: center;
+}
+
+article{
+    display: flex;
+    justify-content: space-around;  
+}
+
+article div{
+    text-align: center;
+    font-size: 3rem;
+    margin-top: 5rem;
+    margin-bottom: 10rem;
+}
+    
+article img{
+    width:60%;
+}
+
+.devices{
+    width: 100%;
+}
+
+/* This is footer*/
+
+footer {
+    font-size: 2.5rem;
+    text-align: center;
+    margin-top: 5rem;
+}
+
+.sns-icon{
+    display: flex;
+    justify-content: center;
```

Comment:
```suggestion
```
The social media icons are already centred due to the footer's `text-align: center` so we don't need  flexbox here